### ISSUE TITLE
maven-junit-plugin <configuration><links> is no longer valid

### ISSUE THREAD

**Issue by aledell** [mojohaus/mojo-parent#51]
Since 73c10ffe (2015-07-18), mojo-parent's pom.xml has had this in the maven-javadoc-plugin:
```
   <configuration>
     <links>
       <link>http://junit-team.github.io/junit/javadoc/latest/</link>
     </links>
   </configuration>
```
(before that, the <link/> pointed at http://junit.sourceforge.net/javadoc/

That link (up until recently) resolved to the javadoc for junit-4.12.

Recently, junit-team.github.io began redirecting to junit.org (presumably for the junit-5.0.0 release on 2017-09-10).

And that means that "mvn site" on projects like versions-maven-plugin produces an error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.6:site (default-site) on project versions-maven-plugin: Error generating maven-javadoc-plugin:2.10.4:javadoc:
[ERROR] Exit code: 1 - javadoc: warning - Error fetching URL: http://junit-team.github.io/junit/javadoc/latest/
```

The now redirected-to junit.org page has links to junit5 javadoc (e.g., http://junit.org/junit5/docs/current/api/), but given that adoption of junit5 is still very early, I suspect we still want junit4 javadoc in mojo-parent.

Luckily, one can still find the junit4 javadoc at http://junit.org/junit4/javadoc/latest/

**Comment by aledell** [comment]
Interestingly, if one disables DocLint in JDK8 (see [this fix for maven-versions-plugin](https://github.com/mojohaus/versions-maven-plugin/pull/209), the \[ERROR\] about the the invalid link becomes a warning:
```
[WARNING] javadoc: warning - Error fetching URL: http://junit-team.github.io/junit/javadoc/latest/
```

**Comment by stefanbirkner** [comment]
Fixed by #52.
